src/liblvgl/display.c

```diff
@@ -26,9 +26,14 @@ static void disp_daemon(void* ign) {
 	}
 }
 
-static void vex_display_flush(int32_t x1, int32_t y1, int32_t x2, int32_t y2, const lv_color_t* color) {
-	vexDisplayCopyRect(x1, y1, x2, y2, (uint32_t*)color, x2 - x1 + 1);
-	lv_flush_ready();
+static void lvgl_display_flush(int32_t x1, int32_t y1, int32_t x2, int32_t y2, const lv_color_t* color) {
+	if(screen_copy_area(x1, y1, x2, y2, (uint32_t*)color, x2 - x1 + 1) == 1) {
+		lv_flush_ready();
+	}
+	else {
+		errno = PROS_ERR;
```

Comment:
Why is errno being set to PROS_ERR? I don't think this is descriptive or accurate.